tests/stubtest_third_party.py

```diff
@@ -131,28 +131,45 @@ def run_stubtest(
         try:
             subprocess.run(stubtest_cmd, env=stubtest_env, check=True, capture_output=True)
         except subprocess.CalledProcessError as e:
-            print_error("fail")
+            print_error("fail\n")
+            divider = "*" * 70
```

Comment:
Maybe add a print_divider helper that we can reuse in other scripts?

Reply:
Heh, sure